users.json

```diff
@@ -7,5 +7,8 @@
     },
     "rachitGulati": {
         "email": ""
-    }
-}
\ No newline at end of file
+    },
+    "gopalgoel19": {
+        "email": ""
+    },
```

Comment:
Invalid JSON. The person who reviewed this PR should stop programming.

Reply:
True indeed 😂😂 but what on earth make a blunder in 3 lines of small json 😎. We should send that guy in prison 😜